### ISSUE TITLE
scoop install fails for v0.2.5

### ISSUE THREAD

**Issue by ciarancourtney** [David-OConnor/pyflow#53]
```bash
scoop install pyflow
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'pyflow' (0.2.5) [64bit]
Loading pyflow.zip from cache.
Checking hash of pyflow.zip ... ok.
Extracting pyflow.zip ... done.
Linking ~\scoop\apps\pyflow\current => ~\scoop\apps\pyflow\0.2.5
Creating shim for 'pyflow'.
Can't shim 'pyflow.exe': File doesn't exist.
```

Pinning to v0.2.4 works for now:

```bash
scoop install pyflow@0.2.4
WARN  Given version (0.2.4) does not match manifest (0.2.5)
WARN  Attempting to generate manifest for 'pyflow' (0.2.4)
Autoupdating pyflow
Downloading pyflow.zip to compute hashes!
Loading pyflow.zip from cache
Computed hash: 8724de8e6cbb5e4b3eb9653640b920ae08b595c1aebd1eda9df77db49ec4f308
Writing updated pyflow manifest
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'pyflow' (0.2.4) [64bit]
Loading pyflow.zip from cache.
Checking hash of pyflow.zip ... ok.
Extracting pyflow.zip ... done.
Linking ~\scoop\apps\pyflow\current => ~\scoop\apps\pyflow\0.2.4
Creating shim for 'pyflow'.
'pyflow' (0.2.4) was installed successfully!
```

**Comment by ciarancourtney** [comment]
Issue is the `pyflow.zip` for 0.2.5 contains the MSI instead of the binary.

FYI some scoop packages dep on lessmsi if that reduces complexity in managing two windows install options

**Comment by David-OConnor** [comment]
Nice find. Fixed.

**Comment by ciarancourtney** [comment]
You'll need to post fix 0.2.5 or release 0.2.6, the hash check fails now:

```bash
$ scoop update pyflow
pyflow: 0.2.4 -> 0.2.5
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Updating one outdated app:
Updating 'pyflow' (0.2.4 -> 0.2.5)
Downloading new version
Starting download with aria2 ...
Download: [#410ae4 0B/2.9MiB(0%) CN:1 DL:0B]
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 410ae4|OK  |   3.2MiB/s|C:/Users/ciara/scoop/cache/pyflow#0.2.5#https_github.com_David-OConnor_pyflow_releases_download_0.2.5_pyflow.zip
Download: Status Legend:
Download: (OK):download completed.
Checking hash of pyflow.zip ... ERROR Hash check failed!
App:         main/pyflow
URL:         https://github.com/David-OConnor/pyflow/releases/download/0.2.5/pyflow.zip
First bytes: 50 4B 03 04 14 00 00 00
Expected:    da811fff144fb9b7deca5ee69b57316269aa68eca3a3a4b8e5f3c1f1a33e8d5a
Actual:      ce66e10a0154500e27827dc0947e8cc535274f1e5591acfbf7f37809d6660915

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=pyflow%400.2.5%3a+hash+check+failed
```

**Comment by David-OConnor** [comment]
Should work now; re-released as `v0.2.5.1`, with no change from `v0.2.5`.

**Comment by ciarancourtney** [comment]
yep, confirmed 👍
